broker/src/main/java/io/zeebe/broker/system/monitoring/ProcessLifetimeMetrics.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.system.monitoring;
+
+import io.prometheus.client.Gauge;
+
+public class ProcessLifetimeMetrics {
+
+  public static final String ZEEBE_NAMESPACE = "zeebe";
+  public static final String PROCESS_NAME_LABEL = "processName";
+  public static final String STEP_NAME_LABEL = "stepName";
+  private static final Gauge STARTUP_METRIC =
+      Gauge.build()
+          .namespace(ZEEBE_NAMESPACE)
+          .name("broker_process_startup_metric")
+          .labelNames(PROCESS_NAME_LABEL, STEP_NAME_LABEL)
```

Comment:
I think we didn't need the "process name", this is automatically added by Prometheus on which pod this metric was scraped.